.github/CODEOWNERS

```diff
@@ -11,6 +11,7 @@
 /plugins/cost-insights                      @backstage/silver-lining
 /plugins/cloudbuild                         @trivago/ebarrios
 /plugins/search                             @backstage/techdocs-core
+/plugins/search-indexer-backend             @backstage/techdocs-core
 /plugins/techdocs                           @backstage/techdocs-core
```

Comment:
I guess we want to add search-backend here as well

Reply:
⭐